src/components/Atoms/AddTaskButton/index.jsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styled from "styled-components";
+import TEXT from "../../../variables/texts";
+import COLOR from "../../../variables/color";
+import plus from "../../../assets/svg/plus.svg";
+
+const AddTaskButton = ({ onClick }) => {
+  return (
+    <StyledTaskButton onClick={onClick}>
+      <StyledGreenOval></StyledGreenOval>
+      <StyledTask>
+        <img src={plus}></img>

```

Comment:
```suggestion
      <StyledGreenOval/ >
      <StyledTask>
        <img src={plus}/ >
```
Reactでは、子要素を持たない場合このように書くことができます。
（セルフクロージングタグといいます）